### ISSUE TITLE
Hide button image from screen readers

### ISSUE THREAD

**Issue by fredrikdomeij** [TheIdentitySelector/thiss-js#218]
The image in the Standard button is missing an **alt** attribute, and is also missing **aria-hidden=true**, this make screen readers notify users of the image.
This results in screen readers notifying users of the button which in unnecissary.
Resolved by https://github.com/TheIdentitySelector/thiss-js/pull/217.
Other images at the discovery service may have similar issues.
